app/assets/stylesheets/govuk_publishing_components/components/_layout-header.scss

```diff
@@ -103,8 +102,9 @@
 }
 
 .gem-c-header__content.govuk-header__content {
-  width: auto;
   padding-left: 0;
+  padding-bottom: govuk-spacing(2);
```

Comment:
Unfortunately this seems to be affecting vertical spacing on the header (even on the variants without the menu). It's adding some extra space below the logo: 
Before | After
------------ | -------------
<img width="415" alt="Screenshot 2021-03-17 at 14 58 57" src="https://user-images.githubusercontent.com/7116819/111488692-62764f00-8731-11eb-839e-d279d882f6fd.png"> | <img width="417" alt="Screenshot 2021-03-17 at 14 58 36" src="https://user-images.githubusercontent.com/7116819/111488688-61ddb880-8731-11eb-8e74-719c6479785b.png">

Reply:
Ah, bother - this isn't the quick fix I was hoping it was.